### PR TITLE
docs: add issue and feature templates

### PR DESCRIPTION
**Pull request by mkilpatrick**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      description: Any other context or screenshots about the feature request here.
+  - type: checkboxes
+    id: checkboxes
+    attributes:
+      label: Validations
+      description: Before submitting the issue, please make sure you do the following
```
Comment (EmilyZhang777) [inline .github/ISSUE_TEMPLATE/feature_request.yml:37]: same comment about "issue"

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      options:
+        - label: Check that there isn't already an issue that requests the same feature to avoid creating a duplicate.
```
Comment (EmilyZhang777) [inline .github/ISSUE_TEMPLATE/feature_request.yml:39]: Will there be any link for existing requests as well? If so, can we include that here?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for your interest in the project and taking the time to fill out this feature report!
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: Description
+      description: "Clear and concise description of the problem. Please make the reason and usecases as detailed as possible. If you intend to submit a PR for this issue, tell us in the description. Thanks!"
```
Comment (EmilyZhang777) [inline .github/ISSUE_TEMPLATE/feature_request.yml:12]: I think "request" (instead of "issue") would be better, but up to you!

**Review by EmilyZhang777 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (mkilpatrick) [reply .github/ISSUE_TEMPLATE/feature_request.yml:39]: No, there's no list of features page. They're just other issues.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 3 above)
Comment (mkilpatrick) [reply .github/ISSUE_TEMPLATE/feature_request.yml:12]: It's a Github "issue" which is why I have that term.

**Pull request merged**